### PR TITLE
wrote an article on app settings

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

File: content/docs/web-interface/application-settings.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+## Application settings
+
+On each application you will find the settings page.
+In this article we will shortly walk through its content.
+
+### Collaborators
+
+Per application you can manage the permission settings of wercker users, wercker
+offers three types of roles and permissions.
```
Comment (flenter) [inline content/docs/web-interface/application-settings.md]: Maybe explain what they are? Or link to an article that does?

Comment (flenter) [inline content/docs/web-interface/application-settings.md]: The link is provided further down. So I guess my comment is: move that up.

Comment (lindseybateman) [inline content/docs/web-interface/application-settings.md]: check added extra link

File: content/docs/web-interface/application-settings.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+Here you can switch the wercker stack for your application.
+New builds and deploys will use this stack. Want to read more on our infrastructure
+stacks, continue reading the following link.
+
+[Read more on stacks &rsaquo;](/docs/pipelines/stacks.html)
+
+#### API user
+
+This section if only visible when the owner of an application is an organization.
```
Comment (flenter) [inline content/docs/web-interface/application-settings.md]: `is only visible`, not `if only visible`

File: content/docs/web-interface/roles-and-permissions.md [hunk 3]
```diff
@@ -17,9 +17,13 @@ A user with **build + deployment** permissions can do all things a user with **b
 
 ### Admin
 
-Like the owner of a project, users with admin permissions can do the same as with **build + deployment** permissions and also:
+Like the owner of a application, users with admin permissions can do the same as with **build + deployment** permissions and also:
```
Comment (flenter) [inline content/docs/web-interface/roles-and-permissions.md]: `of an application`

File: content/docs/web-interface/application-settings.md (hunk 2 above)
Comment (lindseybateman) [inline content/docs/web-interface/application-settings.md]: check

File: content/docs/web-interface/roles-and-permissions.md (hunk 3 above)
Comment (lindseybateman) [inline content/docs/web-interface/roles-and-permissions.md]: check

**Comment by mies** [comment]
looks  good!

**Pull request merged**